### PR TITLE
Add mark-tick test

### PR DESCRIPTION
**Pull request by YuhanLu**

### CODE REVIEW

File: test/compile/mark-tick.test.ts [hunk 1]
```diff
@@ -12,33 +15,65 @@ describe('Mark: Tick', function() {
...
     it('should scale on x', function() {
-      // TODO
+      assert.deepEqual(props.xc, {scale: X, field: 'Horsepower'});
     });
   });
 
   describe('with quantitative y', function() {
+    const model = parseModel({
+      'mark': 'tick',
+      'encoding': {'y': {'field': 'Cylinders','type': 'ordinal'}},
```
Comment (domoritz) [inline test/compile/mark-tick.test.ts]: y is ordinal here, doesn't match the title

File: test/compile/mark-tick.test.ts [hunk 2]
```diff
@@ -12,33 +15,65 @@ describe('Mark: Tick', function() {
...
+      'data': {'url': 'data/cars.json'}
+    });
+
+    const props = tick.properties(model);
     it('should be centered on x', function() {
-      // TODO
+      assert.deepEqual(props.xc, {value: model.fieldDef(X).scale.bandWidth / 2});
```
Comment (domoritz) [inline test/compile/mark-tick.test.ts]: Let's hard code it.

Comment (YuhanLu) [inline test/compile/mark-tick.test.ts]: but isn't this line of code only been used once?

**Comment by domoritz** [comment]
@YuhanLu Can you push your local changes?

**Comment by kanitw** [comment]
Can you please name commits more reasonably next time? 

For example, 18047fa2e1700244866aa68c894707083b9aa07c looks like a "merge” commit, but you name it "add mark tick test”.  

0d5ccaadb9a96fbfcc2209f7d26dcd5f08255a87 is "modifying test to use hardcode value"

**Comment by domoritz** [comment]
Closing as discussed on slack. @YuhanLu will create a new pr.

**Pull request closed without merging**